### PR TITLE
Add more mask fields to document mask model

### PR DESCRIPTION
**Pull request by Aviv-Galmidi**
Add more mask fields to document mask model. Both to v2 and the new v3

Signed-off-by: Aviv.Galmidi <AvivGalmidi@gmail.com>

### CODE REVIEW

**Comment by bartavelle** [comment]
LGTM

**Comment by bartavelle** [comment]
I don't think I can approve anything :/

File: curiefense/curieconf/utils/curieconf/utils/__init__.py [hunk 1]
```diff
@@ -15,7 +15,6 @@
     "aclprofiles": "config/json/acl-profiles.json",
     "globalfilters": "config/json/globalfilter-lists.json",
     "flowcontrol": "config/json/flow-control.json",
-    "actions": "config/json/actions.json",
```
Comment (Aviv-Galmidi) [inline curiefense/curieconf/utils/curieconf/utils/__init__.py:18]: This was duplicated from [line 10](https://github.com/curiefense/curiefense/pull/889/files#diff-a81b3b8329400c8f24d1e8582293c6c0cdf7bf1f603f71a13b74be84ea932d3dL10)

**Comment by Aviv-Galmidi** [comment]
@tzuryby Could you take a look at this PR and approve if possible?

**Review by tzuryby — APPROVED** [review]

**Pull request merged**